Jeff_pet_codes/ArrayThings/src/StudentGrade.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Created by Jeffries Pc on 24/07/2017.
+ */
+public class StudentGrade {
+
+
+    public String[] splitStudentGrade (){
+    String studentGrades = ("Pass,Fail,Pass,Resit,Pending");
```

Comment:
This is good but I don't understand why you have opening and closing brackets while declaring string studentGrades?